indexservice/indexer.go

```diff
@@ -56,20 +58,24 @@ type (
 		ExecutionHash string
 		BlockHash     string
 	}
-
 	// ActionHistory defines the schema of "action history" table
 	ActionHistory struct {
 		NodeAddress string
 		UserAddress string
 		ActionHash  string
 	}
-
 	// ActionToBlock defines the schema of "action hash to block hash" table
 	ActionToBlock struct {
 		NodeAddress string
 		ActionHash  string
 		BlockHash   string
 	}
+	// HashToReceipt defines the schema of "hash to receipt" table
+	HashToReceipt struct {
```

Comment:
Can we build the receipt to block index instead? This could avoid us from storing the receipt data twice?

Reply:
OK, cool. I originally think we should avoid call data twice

Reply:
but change to receipt_to_block does keep the common pattern of other tables